README.md

```diff
@@ -4,25 +4,23 @@
 [![Discord](https://img.shields.io/badge/chat-on%20discord-brightgreen.svg)](https://discord.gg/GsEFRM8)
 [![Try it on gitpod](https://img.shields.io/badge/try-on%20gitpod-brightgreen.svg)](https://gitpod.io/#https://github.com/PrismarineJS/prismarine-web-client)
 
-### A Minecraft client running in a web page. **Live demo at https://prismarine.js.org/prismarine-web-client/**
-
+A Minecraft client running in a web page. **Live demo at https://prismarine.js.org/prismarine-web-client/**
 
 
 ## How it Works
-prismarine-web-client runs mineflayer and prismarine-viewer in the browser, which connects over WebSocket to a proxy which translates the WebSocket connection into TCP to connect to normal Minecraft servers.
+prismarine-web-client runs mineflayer and prismarine-viewer in the browser, which connects over WebSocket to a proxy 
+which translates the WebSocket connection into TCP to connect to normal Minecraft servers.
```

Comment:
Why a new line here?

Reply:
it's a good practice to avoid going beyond a reasonable number of characters per line (something like 120), so there's no horizontal scrolling
markdown doesn't do anything with new lines at rendering time (it only does a new paragraph when there is an empty line)

Reply:
Interesting. I know GitHub enables soft-wrapping, but you're probably much more experienced writing good documentation than I am.

Reply:
yeah it's not about github, more about IDEs, they usually don't enable soft-wrapping, at least not by default
so it's more convenient if there are not super long lines
(btw this apply to code, commit messages, ...)

Reply:
Hmm. I've just always kept it to one main function per line (with the exception of builder functions). Will keep in mind though.